src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs

```diff
@@ -9,6 +9,19 @@ namespace System.Security.Cryptography.Encryption.Tests.Symmetric
 {
     public static class TrivialTests
     {
+        [Theory]
+        [InlineData(-1)]
+        [InlineData(0)]
+        [InlineData(7)]
+        [InlineData(48)]
+        public static void TestBadFeedbackSize(int size)
+        {
+            using (var t = new Trivial { BlockSize = 40 })
+            {
+                Assert.Throws<CryptographicException>(() => t.FeedbackSize = size);
```

Comment:
How about a test that shows that if `FeedbackSize`  was set as a large/legal value, then block size was reduced, that there's no (immediate) exception or alteration of the FeedbackSize value?

Or that if FeedbackSizeValue was itself set to a non-mod-8 value that the getter happily returns it without throwing.